### PR TITLE
Updating UserRoleRemove to use doExecute()

### PR DESCRIPTION
**Pull request by ndeet**
See d.o. issue https://www.drupal.org/node/2572119

### CODE REVIEW

File: src/Plugin/RulesAction/UserRoleRemove.php [hunk 1]
```diff
@@ -38,15 +39,14 @@ class UserRoleRemove extends RulesActionBase {
...
+   *   Array of user roles.
    */
-  public function execute() {
-
-    /** @var \Drupal\user\Entity\User $account */
-    $account = $this->getContextValue('user');
-
-    /** @var \Drupal\user\RoleInterface $roles */
-    $roles = $this->getContextValue('roles');
+  protected function doExecute(UserInterface $account, $roles) {
```
Comment (klausi) [inline src/Plugin/RulesAction/UserRoleRemove.php]: should type hint to array for $roles

File: src/Plugin/RulesAction/UserRoleRemove.php [hunk 2]
```diff
@@ -38,15 +39,14 @@ class UserRoleRemove extends RulesActionBase {
   protected $saveLater = FALSE;
 
   /**
-   * {@inheritdoc}
+   * Remove role from a user.
+   *
+   * @param \Drupal\user\UserInterface $account
+   *   User object the roles should be removed from.
+   * @param array $roles
```
Comment (klausi) [inline src/Plugin/RulesAction/UserRoleRemove.php]: should use the InterfaceName[] data type syntax.

**Comment by ndeet** [comment]
Fixed.

**Comment by klausi** [comment]
merged, thanks!

**Pull request closed without merging**